### ISSUE TITLE
changeParameterValueAsync with v3

### ISSUE THREAD

**Issue by vdupin** [tableau/embedding-api-v3-guide#24]
Hello,

I have a problem with v3 embedding-api,
It's an Angular project.
I import tableau with:
`<script type="module" src="https://embedding.tableauusercontent.com/tableau.embedding.3.0.0.min.js"></script>`

Here is my html:
`<tableau-viz id="tableau" src="myUrl" toolbar="hidden" hide-tabs> </tableau-viz>`

And my typescript:
```
public ngAfterViewInit(): void {
    const vizTableau: any = document.getElementById('tableau');

    vizTableau.addEventListener('firstinteractive', () => {
        vizTableau.workbook.changeParameterValueAsync('XXX', 'YYY');
    });
}
```

I have the following error: 
ERROR TypeError: vizTableau.workbook.changeParameterValueAsync is not a function

For information, the tableau is well displayed, and i can access to vizTableau.workbook.activeSheet.size.maxSize property for example.

In https://github.com/tableau/embedding-api-v3-guide#things-that-remain-the-same, in "Things that remain the same" section, there is an example of changeParameterValueAsync use.  But it doesn't seem to work in my case. 

Is this supposed to work? (vizTableau.getWorkbook() does not work too)
Do i have to use Initialization via JavaScript in this case ?

Thank you.

**Comment by illonage** [comment]
Hi @vdupin - Thank you for opening this issue. Unfortunately, this function is not implemented yet and should be in v3.2 that is going to be available in the upcoming weeks in Developer Preview.

**Comment by vdupin** [comment]
For information :

```
public ngAfterViewInit(): void {
    const vizTableau: any = document.getElementById('tableau');

    vizTableau.addEventListener('firstinteractive', () => {
        vizTableau.activeSheet.applyFilterAsync(...); // DOESN'T WORK
        vizTableau.workbook.activeSheet.applyFilterAsync(...); // DOESN'T WORK
    });
}
```

 The error is : Cannot read properties of undefined (reading 'applyFilterAsync')

**Comment by ashwinar** [comment]
@vdupin changeParameterValueAsync() is available in v3.2, which is in preview currently. Feel free to go check it out!

**Comment by Mrkbingham** [comment]
Hey @ashwinar are there docs available for the preview?

**Comment by ashwinar** [comment]
Yes, there are. Here's the landing page for developer preview. It has references to the preview library & preview API reference docs:

https://embedding.tableauusercontent.com/preview/getting-started-v3.html

**Comment by hackwithharsha** [comment]
@illonage , any update when Tableau 2022.2 will be available, currently that version is not available to use tableau embedding api v3.2 ?

**Comment by illonage** [comment]
@hackwithharsha it is available now for Tableau Cloud customers. Let us know if you have any issues

**Comment by hackwithharsha** [comment]
@illonage Thank you, any update for tableau server enterprise customers..

**Comment by hackwithharsha** [comment]
@illonage Does this changeParameterAsyncValue available in Server 2021.1.5 ?

**Comment by illonage** [comment]
Not via the Embedding API v3, you could use the [JavaScript API v2](https://help.tableau.com/current/api/js_api/en-us/JavaScriptAPI/js_api.htm).
